base/applications/network/ipconfig/ipconfig.c

```diff
@@ -686,8 +751,19 @@ VOID ShowInfo(BOOL bAll)
 
             if (pAdapter->DhcpEnabled && _tcscmp(pAdapter->DhcpServer.IpAddress.String, _T("255.255.255.255")))
             {
-                _tprintf(_T("\tLease Obtained. . . . . . . . . . : %s"), _tasctime(localtime(&pAdapter->LeaseObtained)));
-                _tprintf(_T("\tLease Expires . . . . . . . . . . : %s"), _tasctime(localtime(&pAdapter->LeaseExpires)));
+                PTSTR DateTimeString;
+                DateTimeString = AllocateLocalizedDateTimeFromTimet(pAdapter->LeaseObtained);
+                _tprintf(_T("\tLease Obtained. . . . . . . . . . : %s\n"), DateTimeString ? DateTimeString : _T(""));
```

Comment:
empty string if failed, or "n/a" ? (same question for below too)

Reply:
In ipconfig, empty string usually indicates the field is really empty. So "N/A" would be better I think.
In the source, nearly no good error handling we can reference, like:
```
LPTSTR lpDesc = GetConnectionDescription(pAdapter->AdapterName);
_tprintf(_T("\tDescription . . . . . . . . . . . : %s\n"), lpDesc);
HeapFree(ProcessHeap, 0, lpDesc);
```
Where `lpDesc` can be `NULL` if failed, and ipconfig will prints null-pointer and frees it

Reply:
committed `"N/A"`